Makefile

```diff
@@ -108,11 +112,21 @@ poetry.lock: pyproject.toml
 	@if [ -z "$(poetry env list)" -o -n "${force}" ]; then poetry sync --without dev; fi
 	@ touch .make-poetry-sync-no-dev
 
+.make-playwright-install: .make-poetry-sync-no-dev
```

Comment:
playwright install is not supported for my OS and fails when I'm trying to run tests. Looks like it supports only ubuntu-based linux distributions https://github.com/microsoft/playwright/issues/23949
```
testsuite (add-ui-testing)❯ make ui flags="--headed --browser chromium --browser firefox"
Installing all Playwright browsers...
BEWARE: your OS is not officially supported by Playwright; installing dependencies for ubuntu20.04-x64 as a fallback.
Installing dependencies...
Switching to root user to install dependencies...
[sudo] password for averevki:
sh: line 1: apt-get: command not found
Failed to install browsers
Error: Installation process exited with code: 127
make: *** [Makefile:116: .make-playwright-install] Error 1
```